[2025-01-19T10:28:17.216+0000] {processor.py:186} INFO - Started process (PID=143) to work on /opt/airflow/dags/tst.py
[2025-01-19T10:28:17.217+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/tst.py for tasks to queue
[2025-01-19T10:28:17.218+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:17.218+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/tst.py
[2025-01-19T10:28:17.232+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:17.232+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/tst.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/tst.py", line 11
    )
    ^
SyntaxError: positional argument follows keyword argument
[2025-01-19T10:28:17.233+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/tst.py
[2025-01-19T10:28:17.247+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/tst.py took 0.035 seconds
[2025-01-19T10:28:20.272+0000] {processor.py:186} INFO - Started process (PID=150) to work on /opt/airflow/dags/tst.py
[2025-01-19T10:28:20.273+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/tst.py for tasks to queue
[2025-01-19T10:28:20.274+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:20.274+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/tst.py
[2025-01-19T10:28:20.284+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:20.284+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/tst.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/tst.py", line 11
    )
     ^
SyntaxError: expected ':'
[2025-01-19T10:28:20.285+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/tst.py
[2025-01-19T10:28:20.298+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/tst.py took 0.030 seconds
[2025-01-19T10:28:22.316+0000] {processor.py:186} INFO - Started process (PID=151) to work on /opt/airflow/dags/tst.py
[2025-01-19T10:28:22.317+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/tst.py for tasks to queue
[2025-01-19T10:28:22.319+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:22.318+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/tst.py
[2025-01-19T10:28:22.334+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:22.334+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/tst.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/tst.py", line 11
    )
     ^
SyntaxError: expected ':'
[2025-01-19T10:28:22.335+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/tst.py
[2025-01-19T10:28:22.347+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/tst.py took 0.034 seconds
[2025-01-19T10:28:29.757+0000] {processor.py:186} INFO - Started process (PID=154) to work on /opt/airflow/dags/tst.py
[2025-01-19T10:28:29.758+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/tst.py for tasks to queue
[2025-01-19T10:28:29.760+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:29.760+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/tst.py
[2025-01-19T10:28:29.774+0000] {processor.py:925} INFO - DAG(s) 'tst' retrieved from /opt/airflow/dags/tst.py
[2025-01-19T10:28:29.923+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:29.923+0000] {override.py:1911} INFO - Created Permission View: can read on DAG:tst
[2025-01-19T10:28:29.931+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:29.931+0000] {override.py:1911} INFO - Created Permission View: can delete on DAG:tst
[2025-01-19T10:28:29.941+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:29.941+0000] {override.py:1911} INFO - Created Permission View: can edit on DAG:tst
[2025-01-19T10:28:29.951+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:29.951+0000] {override.py:1911} INFO - Created Permission View: can read on DAG Run:tst
[2025-01-19T10:28:29.956+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:29.956+0000] {override.py:1911} INFO - Created Permission View: can create on DAG Run:tst
[2025-01-19T10:28:29.962+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:29.962+0000] {override.py:1911} INFO - Created Permission View: menu access on DAG Run:tst
[2025-01-19T10:28:29.968+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:29.967+0000] {override.py:1911} INFO - Created Permission View: can delete on DAG Run:tst
[2025-01-19T10:28:29.968+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:29.968+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-19T10:28:29.975+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:29.975+0000] {dag.py:3262} INFO - Creating ORM DAG for tst
[2025-01-19T10:28:29.981+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:29.981+0000] {dag.py:4180} INFO - Setting next_dagrun for tst to 2025-01-19 00:00:00+00:00, run_after=2025-01-19 00:00:00+00:00
[2025-01-19T10:28:29.999+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/tst.py took 0.245 seconds
[2025-01-19T10:28:31.808+0000] {processor.py:186} INFO - Started process (PID=155) to work on /opt/airflow/dags/tst.py
[2025-01-19T10:28:31.809+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/tst.py for tasks to queue
[2025-01-19T10:28:31.810+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:31.810+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/tst.py
[2025-01-19T10:28:31.837+0000] {processor.py:925} INFO - DAG(s) 'tst' retrieved from /opt/airflow/dags/tst.py
[2025-01-19T10:28:31.844+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:31.844+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-19T10:28:31.857+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:31.856+0000] {dag.py:4180} INFO - Setting next_dagrun for tst to 2025-01-19 00:00:00+00:00, run_after=2025-01-19 00:00:00+00:00
[2025-01-19T10:28:31.873+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/tst.py took 0.069 seconds
[2025-01-19T10:28:38.876+0000] {processor.py:186} INFO - Started process (PID=156) to work on /opt/airflow/dags/tst.py
[2025-01-19T10:28:38.877+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/tst.py for tasks to queue
[2025-01-19T10:28:38.880+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:38.879+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/tst.py
[2025-01-19T10:28:38.915+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:38.914+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/tst.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/tst.py", line 12
    )
    ^
SyntaxError: unmatched ')'
[2025-01-19T10:28:38.916+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/tst.py
[2025-01-19T10:28:38.931+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/tst.py took 0.059 seconds
[2025-01-19T10:28:40.919+0000] {processor.py:186} INFO - Started process (PID=157) to work on /opt/airflow/dags/tst.py
[2025-01-19T10:28:40.920+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/tst.py for tasks to queue
[2025-01-19T10:28:40.921+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:40.921+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/tst.py
[2025-01-19T10:28:40.934+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:40.933+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/tst.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/tst.py", line 11
    ) as dag
            ^
SyntaxError: expected ':'
[2025-01-19T10:28:40.935+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/tst.py
[2025-01-19T10:28:40.949+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/tst.py took 0.034 seconds
[2025-01-19T10:28:42.948+0000] {processor.py:186} INFO - Started process (PID=158) to work on /opt/airflow/dags/tst.py
[2025-01-19T10:28:42.949+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/tst.py for tasks to queue
[2025-01-19T10:28:42.951+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:42.951+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/tst.py
[2025-01-19T10:28:42.974+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:42.973+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/tst.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/tst.py", line 12
    
    ^
IndentationError: expected an indented block after 'with' statement on line 7
[2025-01-19T10:28:42.974+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/tst.py
[2025-01-19T10:28:42.987+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/tst.py took 0.043 seconds
[2025-01-19T10:28:53.116+0000] {processor.py:186} INFO - Started process (PID=165) to work on /opt/airflow/dags/tst.py
[2025-01-19T10:28:53.116+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/tst.py for tasks to queue
[2025-01-19T10:28:53.118+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:53.118+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/tst.py
[2025-01-19T10:28:53.168+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:53.161+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/tst.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/tst.py", line 12, in <module>
    task=BashOperator(
         ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 489, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'bash_command'
[2025-01-19T10:28:53.170+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/tst.py
[2025-01-19T10:28:53.187+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/tst.py took 0.076 seconds
[2025-01-19T10:28:56.256+0000] {processor.py:186} INFO - Started process (PID=168) to work on /opt/airflow/dags/tst.py
[2025-01-19T10:28:56.257+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/tst.py for tasks to queue
[2025-01-19T10:28:56.259+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:56.259+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/tst.py
[2025-01-19T10:28:56.299+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:28:56.295+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/tst.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/tst.py", line 12, in <module>
    task=BashOperator(
         ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 489, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'bash_command'
[2025-01-19T10:28:56.300+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/tst.py
[2025-01-19T10:28:56.316+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/tst.py took 0.064 seconds
[2025-01-19T10:29:01.323+0000] {processor.py:186} INFO - Started process (PID=169) to work on /opt/airflow/dags/tst.py
[2025-01-19T10:29:01.324+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/tst.py for tasks to queue
[2025-01-19T10:29:01.326+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:29:01.326+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/tst.py
[2025-01-19T10:29:01.352+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:29:01.349+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/tst.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/tst.py", line 12, in <module>
    task=BashOperator(
         ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 489, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'bash_command'
[2025-01-19T10:29:01.354+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/tst.py
[2025-01-19T10:29:01.368+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/tst.py took 0.049 seconds
[2025-01-19T10:29:02.401+0000] {processor.py:186} INFO - Started process (PID=170) to work on /opt/airflow/dags/tst.py
[2025-01-19T10:29:02.401+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/tst.py for tasks to queue
[2025-01-19T10:29:02.403+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:29:02.403+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/tst.py
[2025-01-19T10:29:02.421+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:29:02.418+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/tst.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/tst.py", line 12, in <module>
    task=BashOperator(
         ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 489, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'bash_command'
[2025-01-19T10:29:02.422+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/tst.py
[2025-01-19T10:29:02.436+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/tst.py took 0.039 seconds
[2025-01-19T10:29:04.447+0000] {processor.py:186} INFO - Started process (PID=171) to work on /opt/airflow/dags/tst.py
[2025-01-19T10:29:04.448+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/tst.py for tasks to queue
[2025-01-19T10:29:04.449+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:29:04.449+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/tst.py
[2025-01-19T10:29:04.461+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:29:04.461+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/tst.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/tst.py", line 14
    )
    ^
SyntaxError: positional argument follows keyword argument
[2025-01-19T10:29:04.462+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/tst.py
[2025-01-19T10:29:04.475+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/tst.py took 0.032 seconds
[2025-01-19T10:29:09.545+0000] {processor.py:186} INFO - Started process (PID=172) to work on /opt/airflow/dags/tst.py
[2025-01-19T10:29:09.545+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/tst.py for tasks to queue
[2025-01-19T10:29:09.547+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:29:09.547+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/tst.py
[2025-01-19T10:29:09.569+0000] {processor.py:925} INFO - DAG(s) 'tst' retrieved from /opt/airflow/dags/tst.py
[2025-01-19T10:29:09.712+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:29:09.712+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-19T10:29:09.723+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:29:09.723+0000] {dag.py:4180} INFO - Setting next_dagrun for tst to 2025-01-19 00:00:00+00:00, run_after=2025-01-19 00:00:00+00:00
[2025-01-19T10:29:09.742+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/tst.py took 0.200 seconds
[2025-01-19T10:29:12.591+0000] {processor.py:186} INFO - Started process (PID=173) to work on /opt/airflow/dags/tst.py
[2025-01-19T10:29:12.591+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/tst.py for tasks to queue
[2025-01-19T10:29:12.593+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:29:12.593+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/tst.py
[2025-01-19T10:29:12.614+0000] {processor.py:925} INFO - DAG(s) 'tst' retrieved from /opt/airflow/dags/tst.py
[2025-01-19T10:29:12.622+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:29:12.622+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-19T10:29:12.634+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:29:12.634+0000] {dag.py:4180} INFO - Setting next_dagrun for tst to 2025-01-19 00:00:00+00:00, run_after=2025-01-19 00:00:00+00:00
[2025-01-19T10:29:12.651+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/tst.py took 0.064 seconds
[2025-01-19T10:29:19.648+0000] {processor.py:186} INFO - Started process (PID=180) to work on /opt/airflow/dags/tst.py
[2025-01-19T10:29:19.649+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/tst.py for tasks to queue
[2025-01-19T10:29:19.651+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:29:19.650+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/tst.py
[2025-01-19T10:29:19.673+0000] {processor.py:925} INFO - DAG(s) 'tst' retrieved from /opt/airflow/dags/tst.py
[2025-01-19T10:29:19.680+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:29:19.680+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-19T10:29:19.692+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:29:19.692+0000] {dag.py:4180} INFO - Setting next_dagrun for tst to 2025-01-19 00:00:00+00:00, run_after=2025-01-19 00:00:00+00:00
[2025-01-19T10:29:19.712+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/tst.py took 0.067 seconds
[2025-01-19T10:29:49.963+0000] {processor.py:186} INFO - Started process (PID=189) to work on /opt/airflow/dags/tst.py
[2025-01-19T10:29:49.964+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/tst.py for tasks to queue
[2025-01-19T10:29:49.965+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:29:49.965+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/tst.py
[2025-01-19T10:29:49.978+0000] {processor.py:925} INFO - DAG(s) 'tst' retrieved from /opt/airflow/dags/tst.py
[2025-01-19T10:29:50.129+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:29:50.129+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-19T10:29:50.140+0000] {logging_mixin.py:190} INFO - [2025-01-19T10:29:50.140+0000] {dag.py:4180} INFO - Setting next_dagrun for tst to 2025-01-19 00:00:00+00:00, run_after=2025-01-19 00:00:00+00:00
[2025-01-19T10:29:50.155+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/tst.py took 0.195 seconds
